### PR TITLE
feat: 【质量红线】拦截了需要默认通知流水线执行人 issue #4906

### PR DESCRIPTION
**Pull request by JamiKX1**
质量红线】拦截了需要默认通知流水线执行人

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=4992) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=4992) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-ci?pullRequest=4992) it.</sub>

File: src/backend/ci/core/quality/biz-quality/src/main/kotlin/com/tencent/devops/quality/service/v2/QualityRuleCheckService.kt [hunk 1]
```diff
@@ -648,29 +649,30 @@ class QualityRuleCheckService @Autowired constructor(
             emailResult.append("<br>")
         }
 
+        val templateCode = PIPELINE_QUALITY_AUDIT_NOTIFY_TEMPLATE_V2
```
Comment (ddlin0719) [inline src/backend/ci/core/quality/biz-quality/src/main/kotlin/com/tencent/devops/quality/service/v2/QualityRuleCheckService.kt:652]: templateCode 的变量可以去掉

File: src/backend/ci/core/quality/biz-quality/src/main/kotlin/com/tencent/devops/quality/service/v2/QualityRuleCheckService.kt [hunk 2]
```diff
@@ -648,29 +649,30 @@ class QualityRuleCheckService @Autowired constructor(
...
-                "title" to "【质量红线拦截通知】你有一个流水线被拦截",
-                "projectName" to projectName,
-                "pipelineName" to pipelineName,
-                "buildNo" to buildNo,
-                "time" to time,
-                "result" to messageResult.toString(),
-                "emailResult" to emailResult.toString(),
-                "url" to url
-            )
+                templateCode = templateCode,
```
Comment (ddlin0719) [inline src/backend/ci/core/quality/biz-quality/src/main/kotlin/com/tencent/devops/quality/service/v2/QualityRuleCheckService.kt:656]: 缩进问题

**Comment by ddlin0719** [comment]
LGTM

**Review by irwinsun — APPROVED** [review]

**Review by ddlin0719 — APPROVED** [review]
LGTM

**Pull request merged**